.github/workflows/ci.yml

```diff
@@ -39,6 +39,27 @@ jobs:
           name: macOS
           path: mac/*Toolchain*.tar.gz
 
+  sign-macos:
```

Comment:
I think I would rather have this be a part of the build-macos job and the signing steps having an `if` to only run on master/tags